src/main/java/com/redhat/red/build/finder/ConfigDefaults.java

```diff
@@ -52,6 +52,8 @@ private ConfigDefaults() {
 
     public static final List<Pattern> EXCLUDES = Collections.unmodifiableList(Arrays.asList(Pattern.compile("^(?!.*/pom\\.xml$).*/.*\\.xml$")));
 
+    public static final String OUTPUT_DIR = "./output";
+
```

Comment:
I think I agree it's best as not a global option. But, as to improving the behavior, I wonder as you say, to avoid cluttering the directory, if it would make sense to create a subdirectory inside there.

I thought the current directory is what made sense as a default, but I was always specifying a directory such that I missed seeing this issue. until after the release.